include/seqan3/alignment/matrix/detail/coordinate_matrix.hpp

```diff
@@ -111,7 +248,7 @@ class coordinate_matrix_iterator
     //!\brief Access the pointed-to matrix coordinate column.
     auto operator*() const
     {
-        return std::views::iota(size_t{0u}, column_size)
+        return std::views::iota(index_t{0}, row_count)
```

Comment:
```suggestion
        return std::views::iota(index_t{}, row_count)
```

Is `std::views::iota` working on simd's?

Reply:
Ah okay this is the scalar version.